EMS/core-bundle/src/Command/Submission/ExportCommand.php

```diff
@@ -26,175 +23,49 @@
-        private readonly MailerService $mailerService,
+        private readonly SubmissionExporter $exporter,
+        private readonly StorageManager $storageManager,
+        private readonly AdminHelper $adminHelper,
     ) {
         parent::__construct();
     }
 
     #[\Override]
     protected function configure(): void
     {
-        $this
-            ->addArgument(
-                self::ARG_FIELDS,
-                InputArgument::IS_ARRAY,
-                'Fields to export in a property accessor format [instance] [name] [locale] [submission_date] [data][email] [data][multi-choice][level_0] [data][multi-choice][level_1]'
-            )->addOption(
-                self::OPTION_FILTER,
-                null,
-                InputOption::VALUE_OPTIONAL,
-                'Expression to filter submissions, e.g. "\'true\' == (data[\'recontact-optin\'] ?? \'false\')". The following variables are available: data (array), instance (string), name (string), locale (string), submission_date (date in the ISO 8601 format)'
-            )->addOption(
-                self::OPTION_FILENAME,
-                null,
-                InputOption::VALUE_OPTIONAL,
-                'Export filename, xlsx or csv formats are supported',
-            )->addOption(
-                self::OPTION_EMAIL_TO,
-                null,
-                InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY,
-                'Email addresses where the export will be sent',
-            )->addOption(
-                self::OPTION_EMAIL_SUBJECT,
-                null,
-                InputOption::VALUE_OPTIONAL,
-                'Email\'s subject',
-                'Submissions export'
-            )->addOption(
-                self::OPTION_EXPORT_FORMAT,
-                null,
-                InputOption::VALUE_OPTIONAL,
-                \sprintf('Format of the export. Supported formats: %s', \implode(', ', SpreadsheetGeneratorServiceInterface::FORMAT_WRITERS)),
-            );
+        $this->addArgument(self::ARGUMENT_CONFIG_FILE, InputArgument::REQUIRED, 'JSON config file (path)');
     }
 
     #[\Override]
     protected function initialize(InputInterface $input, OutputInterface $output): void
     {
         parent::initialize($input, $output);
-        $this->fields = $this->getArgumentStringArray(self::ARG_FIELDS);
-        $this->filter = $this->getOptionStringNull(self::OPTION_FILTER);
-        $this->filename = $this->getOptionStringNull(self::OPTION_FILENAME);
-        $this->emailsTo = $this->getOptionStringArray(self::OPTION_EMAIL_TO, false);
-        $this->subject = $this->getOptionString(self::OPTION_EMAIL_SUBJECT);
-        $this->format = $this->getOptionStringNull(self::OPTION_EXPORT_FORMAT);
+
+        $fileIdentifier = $this->getArgumentString(self::ARGUMENT_CONFIG_FILE);
+        $file = $this->getFile($fileIdentifier);
+
+        $this->exportConfig = ExportConfig::fromJson($file->getContent());
```

Comment:
We should also be able to pass a hash, or just the json string

Like: https://github.com/ems-project/elasticms/blob/HEAD/elasticms-cli/src/Command/Import/AbstractImportCommand.php#L155